docs/intro.rst

```diff
@@ -314,17 +314,17 @@ This is the current list of error and warning codes:
 +------------+----------------------------------------------------------------------+
 | **E3**     | *Blank line*                                                         |
 +------------+----------------------------------------------------------------------+
-| E301       | expected 1 blank line, found 0                                       |
+| E301       | expected 1 blank line, found …                                       |
```

Comment:
Let's just stick to adding the new message. Then let's update these in a separate pull request.